modules/betsy/bc1.glsl

```diff
@@ -0,0 +1,480 @@
+#[compute]
```

Comment:
@akien-mga What's the policy on things like glsl compute shader from Betsy. Normally it should go in the thirdparty but maybe you've got other suggestions.